ansible/roles/debops.dovecot/defaults/main.yml

```diff
@@ -58,10 +59,13 @@ dovecot_deny_users: [ 'root' ]
                                                                    # ]]]
 # .. envvar:: dovecot_mail_location [[[
 #
-# Mailbox location. For maildir set something like ``maildir:~/Maildir``.
+# Mailbox location. For mbox set something like ``mbox:~/mail:INBOX=/var/mail/%u``.
 # For more information about the supported format, check `Dovecot Mail
-# Location <http://wiki2.dovecot.org/MailLocation>`_
-dovecot_mail_location: 'mbox:~/mail:INBOX=/var/mail/%u'
+# Location <https://doc.dovecot.org/configuration_manual/mail_location/>`_
+dovecot_mail_location: 'maildir:{{ "/var/vmail/%d/%n/mailbox"
+                                  if (dovecot__ldap_enabled|d(False))
+                                  else
+                                  "~/Maildir" }}'
```

Comment:
What's the reason for changing the default from mbox to maildir? I personally also use maildir, but that's a breaking change that is not topic related.

Reply:
I wanted to be as close as possible as current dovecot config defaults. Looking at their docs and configs, I chosed to use the MailDir sane default.

Reply:
If we are going with variable renames, I think that might a good moment to change the default to Maildir as well. The `mbox` format is old and I'm not sure about scalability, Maildir seems to be more sane. I wonder how [`dbox` storage format](https://wiki2.dovecot.org/MailboxFormat/dbox) would fare in this case and how the conversion process goes for these changes.